BusinessAdapter.FP/Receiving/MessageReceiveInfo.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Schleupen.AS4.BusinessAdapter.FP.Receiving;
+
+/// <summary>
+/// Contains the result of a message query.
+/// </summary>
+public sealed class MessageReceiveInfo
+{

```

Comment:
Naming der Felder, Methoden wäre besser angelegt an das Statusmodell in der B-Api: Provided, Acknowledged